Scoop/NetworkManager.swift

```diff
@@ -95,8 +95,6 @@ class NetworkManager {
                 case .failure(let error):
                     completion(.failure(error))
                     print(error.localizedDescription)
```

Comment:
I'd say with all error print statements, it helps to standardize the output and make sure to specify the cause **then** print the localized description, try not to switch up the order